src/configs/container.rs

```diff
@@ -8,6 +8,7 @@ pub struct ContainerConfig<'a> {
     pub symbol: &'a str,
     pub style: &'a str,
     pub disabled: bool,
+    pub use_container_name: bool,
```

Comment:
Perhaps call this `prefer_container_name` instead, your code seems to include a fallback.